web/src/components/netflow-traffic.tsx

```diff
@@ -387,6 +388,13 @@ export const NetflowTraffic: React.FC<{
     topologyOptions.groupTypes
   ]);
 
+  const getFetchFunctionsClbk = React.useCallback(() => {
```

Comment:
I'm not a huge fan of this `Clbk` suffix here. Can we either:
- use distinct names in `back-and-forth.ts` / `netflow-traffic.tsx`
- rename import like 
```
import { getFetchFunctions as getBackAndForthFetch } from '../utils/back-and-forth';
```

Reply:
done